### DISCUSSION TITLE
Embed tweets inside of issue description and comments

### DISCUSSION THREAD

**Discussion by daniloab** [community/community#12063] [Projects and Issues]
I would like to know if has a way to embed a Twitter in an issue description or comment like this one below:

https://twitter.com/daniloab_/status/1496932406653292550

If don't have yet could be a cool feature to be added to GitHub.

**Comment by heidihelen** [comment]
+1. I would like to be able to embed Tweets in a GitHub issue. When I file feature requests, I link to Tweets, and it would be great if this could be displayed better. Currently, using the embed code from Twitter doesn’t seem to be supported.
